### PR TITLE
VNDA - Algolia

### PR DESCRIPTION
**Pull request by gabrielMatosBoubee**

### CODE REVIEW

File: algolia/loaders/product/vndaListingPage.ts [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+        sortFacetValuesBy: props.sortFacetValuesBy,
+      },
+    },
+  ]);
+
+  const [
+    { hits, page, nbPages, nbHits, hitsPerPage },
+    { facets },
+  ] = results as SearchResponse<VNDAProduct>[];
+  console.log({facets})
```
Comment (guitavano) [inline algolia/loaders/product/vndaListingPage.ts:229]: -

**Comment by lucis** [comment]
What's the motivation of Algolia app "knowing" about the VNDA platform?

**Comment by tlgimenes** [comment]
We should think better on how to integrate these. Algolia knowing vnda seems a bit confusing. Maybe move these loaders to the site it's being used or create a new app for this

**Comment by matheusgr** [comment]
Closed by [tlgimenes](https://github.com/tlgimenes) suggestion.

**Pull request closed without merging**